### DISCUSSION TITLE
Do analytics work in streamlit forms?

### DISCUSSION THREAD

**Discussion by rfsmart-jswingle** [444B/streamlit-analytics2#113] [General]
I have a st.form that contains a st.selectbox and a form button, but it seems that the analytics are not recording the number of times that each of the items in the selectbox are being chosen. Instead the count of the default selected item keeps incrementing instead. Has anyone else experienced this issue or used streamlit-analytics2 successfully to track the components of a form?

**Comment by 444B** [comment]
Hey! Thansk for the question
Can you share a full code example that can demonstrate this? I am not able to see the issue so clearly in my mind from the description
